traits-stubs/setup.py

```diff
@@ -10,16 +10,29 @@
 
 import setuptools
 
+
+def get_long_description():
+    """ Read long description from README.rst. """
+    with open("README.rst", "r", encoding="utf-8") as readme:
+        return readme.read()
+
+
 if __name__ == "__main__":
     setuptools.setup(
         name="traits-stubs",
-        version="0.1.0",
-        description="type annotation integration stubs for traits",
+        version="6.1.0",
```

Comment:
Does `traits-stubs` version have to match `traits`'s?  6.1.0 looks somewhat strange as there are no versions 0-5 for `traits-stubs`.  Alternatively we can specify a minimum version for traits in `install_requires`?

Reply:
So we need to communicate which version of Traits these are the stubs for _somehow_, and the version number of `traits-stubs` itself seems like as good an approach as any.

We could add the Traits minimum version in `install_requires`, too.

Reply:
Maybe the description is a good place? 

If we make a bug fix release traits 6.1.1, we may not need to release `traits-stubs` again (depending on the bug being fixed, of course).

Having the release version obviously different makes it more obvious that the release versions do not need to match: So I don't need to worry even if I have `traits-stub` x.y.z running along with `traits` a.b.c (x.y.z != a.b.c) as long as the install_requires is satisfied.  That said, I am also fine with matching them.

Reply:
Yes; I'd propose that we keep the major.minor versions aligned, but not the bugfix versions. (We could also conceivably want to make bugfix releases of the stubs without a corresponding bugfix release of Traits.)

But with the way that we're currently working, the stubs are closely tied to the Traits version - when we add something to add Traits, we add the corresponding stub; when we delete something from Traits, we delete the corresponding stub.

Consequently there's no forward or backward compatibility outside bugfix versions: traits-stubs 6.1.x will only work with Traits 6.1.x, and traits-stubs 6.2.x will only work with traits 6.2.x. It's not like a normal dependency relationship.

Reply:
Looking at other projects on PyPI, some projects do align with the package version (django-stubs, for example), some don't. Given that we're maintaining and releasing the stubs (as opposed to some 3rd party producing the stubs as a convenience), I think using matching major.minor versions is the way to go.

Reply:
> django-stubs, for example

Ooops, no it doesn't. Bad example. Try https://pypi.org/project/PyQt5-stubs/ instead.